db/schema.rb

```diff
@@ -38,4 +40,27 @@
   end
 
   add_foreign_key "application_version", "application", name: "application_version_application_id_fkey"
+
+  create_view "all_events", sql_definition: <<-SQL
```

Comment:
Do we need these create statements and the new schema now that we have the migrations and other SQL files?

I thought everything was going to be part of the public schema.

Reply:
So in Rails the `schema.rb` contains a full snapshot of what the database structure should be. The migrations tell the story of how we got to the current snapshot, and how to get from a previous state to the current one. scenic adds a layer of extra redundancy because instead of altering views, it deletes and recreates them, and keeps a record of every version of every view. So it does mean every view definition in the schema file will be duplicated in db/views/myview_v[normally_latest].sql, but I think this redundancy is fine, as we've got Rails/scenic working to ensure that the right things are synced in the right ways.